src/components/user/notification.tsx

```diff
@@ -62,152 +94,167 @@ export function Notification({
   }
 
   function getSubscriptionId() {
-    switch (event.type) {
-      case NotificationEventType.SetThreadState:
-      case NotificationEventType.CreateThread:
-        return event.thread
-
-      case NotificationEventType.CreateComment:
+    switch (event.__typename) {
+      case 'SetThreadStateNotificationEvent':
+      case 'CreateThreadNotificationEvent':
+      case 'CreateCommentNotificationEvent':
         return event.thread.id
 
-      case NotificationEventType.CreateEntity:
-      case NotificationEventType.SetLicense:
-      case NotificationEventType.CreateLink:
-      case NotificationEventType.RemoveLink:
-        return event.entity
-
-      case NotificationEventType.CreateEntityRevision:
-      case NotificationEventType.CheckoutRevision:
-      case NotificationEventType.RejectRevision:
-        return event.repository.id
+      //TODO: Check if it's linked to the child
+      case 'CreateEntityLinkNotificationEvent':
+      case 'RemoveEntityLinkNotificationEvent':
+      case 'CreateTaxonomyLinkNotificationEvent':
+      case 'RemoveTaxonomyLinkNotificationEvent':
+        return event.child.id
 
-      case NotificationEventType.CreateTaxonomyAssociation:
-      case NotificationEventType.RemoveTaxonomyAssociation:
+      case 'CreateEntityNotificationEvent':
+      case 'CreateEntityRevisionNotificationEvent':
         return event.entity.id
 
-      case NotificationEventType.SetUuidState:
-        return event.uuid
+      case 'SetLicenseNotificationEvent':
+      case 'CheckoutRevisionNotificationEvent':
+      case 'RejectRevisionNotificationEvent':
+        return event.repository.id
+
+      case 'SetUuidStateNotificationEvent':
+        return event.object.id
 
       default:
         return undefined
     }
   }
 
-  function renderTitle() {
-    switch (event.type) {
-      case NotificationEventType.SetThreadState:
+  function renderText() {
+    switch (event.__typename) {
+      case 'SetThreadStateNotificationEvent':
         return (
           <>
-            <UserLink user={event.actor} /> hat Thread {event.thread.id}
+            {renderUser(event.actor)} hat einen {renderThread(event.thread.id)}{' '}
             {event.archived ? 'archiviert' : 'unarchiviert'}
           </>
         )
```

Comment:
I tried to make this easy to read, okay for now.

@inyono do you have an idea of how we could make this translatable?
My best guess so far is to write it like `"%USER% has %ACTION% a %THREAD%"` and write a little parser, which seems ridiculously complicating for what it is.

Reply:
(the render… functions return links as JSX elements

Reply:
That is usually what an i18n library would handle for us 😛 Otherwise, we'll probably need some parser for that, yeah.

Reply:
This is how it would work with next-i18next
``` jsx 
<Trans
  i18nKey="myKey"
  defaults="hello <italic>beautiful</italic> <bold>{{what}}</bold>" // optional defaultValue
  values={{ what: 'world'}}
  components={{ italic: <i />, bold: <strong /> }}
/>
```
[(source)](https://react.i18next.com/latest/trans-component#alternative-usage-v-11-6-0)

Wrote a little parser anyway, see next comment.